### PR TITLE
Overview style upgrades

### PR DESCRIPTION
**Pull request by lucaspopp0**

### CODE REVIEW

File: services/frontend/src/components/overview/overview-section-cell.js [hunk 1]
```diff
@@ -2,92 +2,65 @@ import React from 'react';
...
-    dayjs,
-    constants: { DB_TIME_FORMAT },
-} = dateUtils;
-
-export class OverviewSectionCell extends React.Component {
+class OverviewSectionCell extends React.Component {
     constructor(props) {
         super(props);
 
         this.getTimingString = this.getTimingString.bind(this);
```
Comment (prao3) [inline services/frontend/src/components/overview/overview-section-cell.js:14]: Im getting an error here: cannot read property 'bind' of undefined

**Comment by prao3** [comment]
image
also getting a weird graphical glitch on text boxes that aren't filled out

File: services/frontend/src/components/overview/overview-section-cell.js (hunk 1 above)
Comment (prao3) [reply services/frontend/src/components/overview/overview-section-cell.js:14]: for line 14 btw

**Review by beasterbro — APPROVED** [review]
we need this now

**Pull request merged**